### ISSUE TITLE
Incorrect points wording on Parch

### ISSUE THREAD

**Issue by KingSupernova31** [KaiGeffen/Celestial#138]
"Worth +1" normally means "this card's point value is changed", but on Parch it's a separate instruction to gain points later. I propose the following text instead:

"Discard each of your cards later in the story, and gain that many points."

**Comment by KaiGeffen** [comment]
Yeah good catch. Could I suggest that it be consistent with Secretary Bird:
`Worth +1 point for each of you cards later in the story. Discard those cards.`
The animation for it is also non-existant (There isn't a move-from-story-to-discard animation).

**Comment by KingSupernova31** [comment]
Ah yeah, I like that even better.
